01_data_exploration.ipynb

```diff
@@ -10,17 +10,17 @@
     "# 1. Data Exploration\n",
```

Comment:
<p>I'm not sure what you mean about using the frequency distribution for Y, could you elaborate?</p>

Reply:
<p>To elaborate: in this case the classes are imbalanced, so the counts for no rotors read into the 2000s while the for rotors data the counts are in the 40s, I thought since a comparison is being made, using the "percentage of the data that is of this histogram bin" would help for comparison.</p><p> </p><p>Now that I look at it though, the plots scale anyway, so the plots would look exactly the same. </p>